packages/strapi-provider-email-nodemailer/README.md

```diff
@@ -0,0 +1,136 @@
+# strapi-provider-email-nodemailer
```

Comment:
README needs to be reconstructed over to the similar providers:
https://github.com/strapi/strapi/blob/master/packages/strapi-provider-email-amazon-ses/README.md
https://github.com/strapi/strapi/blob/master/packages/strapi-provider-email-mailgun/README.md
https://github.com/strapi/strapi/blob/master/packages/strapi-provider-email-sendmail/README.md
as examples